assets/js/misc.js

```diff
@@ -0,0 +1,24 @@
+window.addEventListener("DOMContentLoaded", () => {
```

Comment:
this file exclusively handles tabBars, so it should have a name that reflects that.